code/modules/clothing/clothing.dm

```diff
@@ -453,6 +474,40 @@ BLIND     // can't see anything
 	if(!lavaland_equipment_pressure_check(T))
 		. *= high_pressure_multiplier
 
+/obj/item/clothing/emp_act(severity)
+	if(tracking_bug)
+		tracking_bug.emp_act(severity)
+	. = ..()
+
+/obj/item/clothing/Click(location, control, params = "" as text)
+	if(!iscarbon(usr))
+		return ..()
+	if(!tracking_bug)
+		return ..()
+	var/mob/living/carbon/user = usr
+	if(!in_range(src, user))
+		return ..()
+	if(!user.has_active_hand())
+		return ..()
+	var/atom/held_item = user.get_active_held_item()
+	if(held_item)
+		if(src != held_item || istype(held_item, /obj/item/video_bug))
+			return ..()
+	var/list/click_params = params2list(params)
+	if(length(click_params) && click_params["screen-loc"])
+		var/found_x = abs(15 + tracking_bug.x_coord - text2num(click_params["icon-x"]))
+		var/found_y = abs(15 + tracking_bug.y_coord - text2num(click_params["icon-y"]))
+		if(found_x >= 0 && found_x <= 2 && found_y >= 0 && found_y <= 2)
+			to_chat(user, "<span class='notice'>Your fingers find something stuck to \the [src], and you try to rip it away!</span>")
+			if(!(src in user.do_afters))
+				if(do_after(user, 4 SECONDS, src))
+					to_chat(user, "<span class='notice'>You dislodge \the [tracking_bug] that was planted on \the [src]!</span>")
+					tracking_bug.forceMove(drop_location())
+					user.put_in_hands(tracking_bug)
+					tracking_bug.detach_from_clothing(user, src)
+			return COMPONENT_NO_ATTACK
+	return ..()
```

Comment:
Isn't this handled by pre_attaack of the video bug? Overriding click is horrible